src/plugins/index.js

```diff
@@ -11,5 +11,6 @@ module.exports = {
     prReadyToMerge: require("./pr-ready-to-merge"),
     needsInfo: require("./needs-info"),
     triage: require("./triage"),
-    recurringIssues: require("./recurring-issues")
+    recurringIssues: require("./recurring-issues"),
+    releaseMerge: require("./release-merge-purifier")
```

Comment:
Nitpick: Could we call this something like `release-monitor`? The name `release-merge-purifier` is a bit confusing to me.

Reply:
Oh yeah.... forgot to change it here and the file name.